parseData.py

```diff
@@ -126,14 +126,21 @@ def readSerial(ser,data):
         #No packet detected
         else: i+=1
     print(data)
-
+    currentDateTime = time.strftime("%Y-%m-%dT%H-%M-%S")
+    currentDateTime = 'Logs/ParseDataLog' + currentDateTime + '.txt'
 
 if __name__ == "__main__":
     ser = None
     data = AvionicsData()
     while(True):
         port = input('Enter a Serial Port to connect to:') #Linux: /dev/ttyUSBx, Windows: COMx
         ser = serial.Serial(port, 9600, timeout=0)
+        logFile = input('Do you want to see the current logging of data Yes(Y) or No(N)')
+        if logFile == 'Y':
+            f = open(currentDateTime, "w")
+            f.write(data)
+        if logFile == 'N':
+            print('Script has finished running')
 
         while(ser!=None):
             time.sleep(1)
```

Comment:
In order to determine if we log to a file, we need to pass the file path to the `readSerial(...)` function:
```py
readSerial(ser, data, logFilePath if (logToFile) else None)
```

Reply:
need to change the `readSerial(...)` definition to `readSerial(ser, data, logFilePath)`